src/functions/private/Auth/DeviceFlow/Test-GitHubAccessTokenRefreshRequired.ps1

```diff
@@ -26,7 +26,7 @@
     }
 
     process {
-        $tokenExpirationDate = $Context.TokenExpirationDate
+        $tokenExpirationDate = $Context.TokenExpiresAt
         $remainingDuration = [datetime]$tokenExpirationDate - [datetime]::Now
```

Comment:
Think we can combine these two lines. Just use $Context.TokenExiresAt - [datetime]::Now.